_includes/footer.html

```diff
@@ -36,8 +37,8 @@ <h3 class="h3-mktg mb-3">{{ t.footer.subscribe.heading }}</h3>
                     class="form-input required email d-block col-10 mx-auto py-2 px-3 mb-3" id="mce-EMAIL"
                     autocomplete="home email">
                   <input type="checkbox" value="1" name="group[9617][1]" id="mce-group[9617]-9617-0" checked="checked"
-                    style="display:none">
-                  <input type="submit" value="{{ t.footer.subscribe.button }}" name="subscribe"
+                    style="display:none
```

Comment:
Or you change the PR to draft status if you're still working on it, but also check locally because it's a bug.
```suggestion
                    style="display:none">
```